packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,92 @@
+# Select
+
+## Component Description
+
+Select is a list in which the selected item is always visible while other items are visible on demand by clicking a dropdown button.
+
+Select uses the browser native for the options list. Use a Select when the user can only select one option and there is no need for formatted text.
```

Comment:
This is not accurate if we are inheriting from FAST.

Reply:
I was wondering about this. The Fluent implementation uses the `option` element and I saw that is a valid choice in FAST as well:
> Note: While `<fast-option>`, `<option>`, and elements with a role="option" will all function, `<fast-option>` has built-in accessibility and form association support when used with a `<fast-select>`.

Does this mean that the native `option` element won't have built-in accessibility and form association support?

Reply:
> Does this mean that the native option element won't have built-in accessibility and form association support?

No, but I wonder about how you will encapsulate the styles with a native element. Will you only style options via the ::slotted selector?

Reply:
I might need to spin up a quick prototype, but I thought styling the `option` elements were out of scope and we would use the default browser styling as Fluent does:
![image](https://user-images.githubusercontent.com/49389/230672374-4696d188-68fd-47dc-9595-d83c3e1a19d0.png)

But that might not be what you're asking. Are you implying that without using something like `fast-option`, engineers can arbitrarily style the `option` elements?

Reply:
Is the fluent “style” that options should take on whatever the current UA styling is? If so, we should be able to leverage option elements. But if we need consistent styling across browser we should use the option WC to encapsulate styling and get consistency across browsers. FWIW, across WebXT this is not the visuals I’ve seen. But perhaps it’s what Fluent has decided across all instances.

Reply:
It would be great to see the spec and understand the “why” so we can communicate effectively to teams who have different requirements.

Reply:
From what I gathered when I read through the design spec, Fluent wants to use the native `option` element. I'm not sure what WebXT's requirements are as that wasn't in the Figma spec:
![image](https://user-images.githubusercontent.com/49389/230675529-54e3c479-9c70-4cec-be17-632e32c4dd2a.png)

![image](https://user-images.githubusercontent.com/49389/230675578-9a539be7-b1ae-47f5-8b75-b7b80efd0257.png)

Reply:
The fallout to dropdown here is odd - dropdown isn’t semantic, what is the delta between these two? We need to dig in here about what these two are and how they are different.

Reply:
IMHO the Fluent Dropdown resembles more of the FAST select component, including support for styled items. This also includes support for complex options that include things like icons. Here's the page for Dropdown in the Fluent docs: https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-dropdown--default And select: https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-select--default

I didn't see any guidance for using one or the other. Maybe something we can discuss with @miroslavstastny?